modules/shortcodes/polldaddy.php

```diff
@@ -105,16 +185,20 @@ function polldaddy_shortcode( $atts ) {
 
 			$item_id = esc_js( $item_id );
 
+			if ( is_ssl() )
+				$rating_js_file = "https://polldaddy.com/js/rating/rating.js";
+			else
+				$rating_js_file = "http://i0.poll.fm/js/rating/rating.js";
+
 			if ( $inline ) {
 				return <<<SCRIPT
 <div class="pd-rating" id="pd_rating_holder_{$rating}{$item_id}"></div>
 <script type="text/javascript" charset="UTF-8"><!--//--><![CDATA[//><!--
 PDRTJS_settings_{$rating}{$item_id}={$settings};
 //--><!]]></script>
-<script type="text/javascript" charset="UTF-8" src="http://i.polldaddy.com/ratings/rating.js"></script>
+<script type="text/javascript" charset="UTF-8" src="{$rating_js_file}"></script>
```

Comment:
Couldn't hurt, but in this case we should be using `esc_url()`, shouldn't we? Anyway, thanks, will fix!